jest.config.js

```diff
@@ -0,0 +1,188 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
```

Comment:
This file was generated by `jest --init`. The uncommented settings are the ones worth looking at.